src/components/Navbar.tsx

```diff
@@ -53,18 +55,33 @@ const Navbar = () => {
       <div
         className={
           dropdownVisible
-            ? "absolute right-0 top-[60px] z-20 flex flex-col items-center gap-[20px] border-b border-l            border-r border-dark-tan bg-med-tan p-[20px] sm:hidden "
-            : "align-center hidden flex-row gap-[20px] pr-[40px] sm:flex"
+            ? "md2:hidden absolute right-0 top-[60px] z-20 flex flex-col items-center gap-[20px] border-b border-l border-r border-dark-tan bg-med-tan p-[20px] "
+            : "align-center md2:flex hidden flex-row gap-[30px] pr-[40px]"
         }
       >
-        <Link href="/">
-          <div className="m-auto font-serif text-lg font-medium duration-150 hover:-translate-y-0.5">
-            About Us
+        <Link href="/public/team" onClick={() => setDropdownVisible(false)}>
+          <div className="font-merriweather m-auto font-black text-dark-teal duration-150 hover:-translate-y-0.5 sm:text-sm md:text-lg">
+            Meet TLP
+          </div>
+        </Link>
+
+        <Link href="/public/about" onClick={() => setDropdownVisible(false)}>
+          <div className="font-merriweather m-auto font-black text-dark-teal duration-150 hover:-translate-y-0.5 sm:text-sm md:text-lg">
```

Comment:
I think `font-black` might be a bit too much. Maybe `font-bold` looks better visually. What do you guys think?